packages/components/psammead-grid/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "psammead-grid",
+  "version": "1.0.0-alpha.0",
+  "description": "Grid setup for components",
+  "main": "dist/index.js",
+  "module": "esm/index.js",
+  "sideEffects": false,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/bbc/psammead/tree/latest/packages/components/psammead-grid"
+  },
+  "author": {
+    "name": "Psammead Maintainers",
+    "email": "PsammeadMaintainers@bbc.co.uk"
+  },
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/bbc/psammead/issues"
+  },
+  "keywords": [
+    "bbc",
+    "grid",
+    "cssgrid",
+    "styled-components"
+  ],
+  "dependencies": {
+    "@bbc/gel-foundations": "^3.4.0",
+    "@bbc/psammead-styles": "^2.3.0"
+  },
+  "peerDependencies": {
+    "styled-components": "^4.3.2"
+  }
```

Comment:
We need to add the `alpha` tag.
```
  "publishConfig": {
    "tag": "alpha"
  }
```

Reply:
From the readme:

> Currently this component is in alpha. This is because it has not yet been tested with various assistive technologies. After it has had an accessibility swarm, this will be published under a standard version.

Reply:
For now, it should be published as an alpha.